special-interest-groups/sig-scheduling/roles-and-organization-management.md

```diff
@@ -0,0 +1,64 @@
+# SIG Scheduling Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Scheduling SIG is to have the **Active Contributor** community role in the PD repository.
```

Comment:
Once a people contributes to a part of scheduling system(triage issue, merged a pull request, etc.), he's joined the SIG automatically.

Reply:
done